docs/getting_started.md

```diff
@@ -61,7 +61,7 @@ Visit the [Integrations Autodiscovery][9] page for details about how to configur
 
 #### Containers registry
 
-To change the default registry ([gcr.io/datadoghq][6]) to another registry, use the option `spec.registry`.
+To change the default registry ([gcr.io/datadoghq][6]) to another registry, use the option `spec.global.registry`.
```

Comment:
github ui won't let me make a suggestion for line 93, but the the gcr link is broken: `ttps://gcr.io/datadoghq`